modules/scrabble.js

```diff
@@ -49,11 +49,12 @@ function scoreLetter (letter, str) {
 function computeWord (str) {
   let score = scrabbleScore(str)
   let isImpossible = str.length > 15 || str.length < 2 || /[^a-zA-Z]/.test(str) || score === null
-  let isAllowable = sowpods.verify(str)
+  let formatted = scrabbleNotate(str)
+  let isAllowable = formatted.charAt(formatted.length - 1) === '*'
```

Comment:
Working backwards to see if it's allowable by working out if we've previously found it allowable by seeing if we've added an asterisk is a bit strange.

Reply:
It could be a pointless microoptimisation, but I didn't want to repeat the relatively expensive `sowpods.verify` function more than necessary. I could have `scrabbleNotate` return an object or something, but that makes it more of a pain to map over things. What would you do?

Reply:
Maybe you could compute `isAllowable`, then pass the entire word object through to `scrabbleNotate()`, where you'd be able to access all the various properties and notate it accordingly.